src/brevitas_examples/imagenet_classification/ptq/README.md

```diff
@@ -0,0 +1,86 @@
+- Bias are quantized to int32 values for FP scale factors, int16 or int32 otherwise;
+- Activations quantizer can be symmetric or asymmetric;
+- Three different percentiles can be used for the activations' statistics computation (99.9, 99.99, 99.999).
+
+Furthermore, Brevitas supports the following PTQ techniques:
+- Bias Correction[<sup>1 </sup>];
+- Graph Equalization[<sup>1 </sup>];
+- If Graph Equalization is enabled, it  is possible to use the _merge\_bias_ technique.[<sup>2 </sup>] [<sup>3 </sup>].
+
+It must be noted that Brevitas allows the user to define several other parameters, such as:
+- The Bias Quantization type (Internal scaling vs External scaling);
+- The bit width for every compute layer type (including the bias bit width) and for every activation type;
+- The quantization configuration of specific layers (e.g., it is possible to define a different bit width only for the first and last compute layers);
+- Several possible options for computing the scale factors of compute layers and activations.
+
+For more information about what are the currently supported quantized layers in Brevitas, check the [following file](https://github.com/Xilinx/brevitas/blob/dev/src/brevitas/graph/quantize.py),
+where we map the torch compute layers and activations with their corresponding quantized version.
+
+## Benchmark flow
+
+Starting from pretrained floating-point torchvision models, Brevitas offers the possibility to automatically obtain the corresponding quantized model leveraging torch.fx transformations.
+For the selected subset of torchvision models, we test several possible combinations of the options described above.
+
+The second set of benchmarks will run two pre-defined quantized model definitions, and load the corresponding pre-trained FP weights[<sup>4 </sup>].
+The pre-defined quantized models use floating point scale factors, with a mix of per-tensor and per-channel strategies. Weights and Activations are quantized at 8 bit.
+
+To run the PTQ Benchmark suite on ImageNet:
+
+ - Make sure you have Brevitas installed and the ImageNet dataset in a Pytorch friendly format.
+ - Run the script passing the as `--git-hash` argument a string that will be added to the `RESULT.md` file.
+
+For example, to run the script on the GPU 0:
+```bash
+brevitas_ptq_imagenet_benchmark --imagenet-dir /path/to/imagenet --gpu 0 --git-hash xxxxx
+```
+The script assumes the presence of a `/path/to/imagenet/train` folder, from which the calibration samples will be taken (configurable with the `--calibration-samples` argument), and
+a `/path/to/imagenet/val` that will be used for validation.
+
+After launching the script, a `RESULT.md` markdown file will be generated two tables correspoding to the two types of benchmarks flows.
+
+
+## Export flow
+
+This flow allows to manually specify a pre-trained torchvision model to quantize with the desired quantization configuration, and eventually export it
```

Comment:
1) import torchvision 2) preprocess and apply graph quantization 3) apply PTQ 4) validation accuracy 5) (optional) export